src/ReactiveDomain.Foundation/ReactiveDomain.Foundation.csproj

```diff
@@ -13,5 +13,5 @@
   <ItemGroup>
     <ProjectReference Include="..\ReactiveDomain.Messaging\ReactiveDomain.Messaging.csproj" />
     <ProjectReference Include="..\ReactiveDomain.Persistence\ReactiveDomain.Persistence.csproj" />
-  </ItemGroup>
```

Comment:
It looks like this is the only change in this file, so let's revert it to avoid mucking up the history.